src/engine/ruleset/decoders/f5/f5-apm.yml

```diff
@@ -135,7 +148,8 @@ normalize:
       - rsa.misc.action: +a_append/$~tmp.rsa.misc.action
 
   # 01490007
```

Comment:
I would also remove these comments with the code, as it is explicitly checked on the `check` condition